app/schemas/models/cinematic.schema.js

```diff
@@ -65,6 +65,8 @@ const DialogNode = c.object({
 }, {
   speaker: c.shortString({ enum: ['left', 'right'], title: 'Speaker', description: 'Which character is speaking. Used to select speech bubble.' }),
   text: { type: 'string', title: 'Text', description: 'html text', maxLength: 500 },
+  textAnimationLength: c.int({ title: 'Text Animation Length(ms)', description: 'The number of milliseconds it takes for the text to animate out. Defaults to 1000ms.' }),
+  speakingAnimation: c.shortString({ title: 'Speaking Animation', description: 'The animation to play on the lank while the text is being animated.' }),
```

Comment:
Is this considered a slug?  If so maybe call this `speakingAnimationSlug`?

Reply:
If not maybe `speakingAnimantionName`?  From what I learned in the docs this references a named animation on the Thang (Lank?) and it would be helpful to call that out in the name.

Reply:
Excellent call out. On the lank these are called actions so `speakingAnimationAction` is a much better name.